addons/web/static/src/public/colibri.js

```diff
@@ -10,6 +10,8 @@ let Markup = null;
 export const SKIP_IMPLICIT_UPDATE = Symbol();
 
 export class Colibri {
+    static INITIAL_VALUE = Symbol("initial value");
```

Comment:
i guess i would make that a simple constant (but keep exporting it on Interaction)
```js
export const INITIAL_VALUE = Symbol("initial value");
```
so people cannot mess with it.